Documentation/subcommands/api-service.md

```diff
@@ -0,0 +1,19 @@
+# rkt api-service
+
+## Overview
+
+The API service is designed to help users to get a well defined and structual result for listing and introspecting their pods and images.
+The API service is implemented with [gRPC](http://www.grpc.io/).
+The API service is designed to run without root privileges, and currently provides a read-only interface.
+The API service is optional for running pods, the start/stop/crash of the API service won't affect any pods or images.
+
+## Running the API service
+
+The API service is implemented by the `rkt api-service` command.
+When started, it will listen on the URL provided by the `--listen-client-url` flag for incoming gRPC requests.
```

Comment:
@joshix I feel we'd better listen on `127.0.0.1` by default so that it won't accept any incoming requests from outside by default?